embedded/drivers/README.md

```diff
@@ -31,3 +31,6 @@ int read_data(uint8_t *buff, int size); //Read data from a device
 uint8_t read_from_reg(uint8_t reg); // Read data from a specific register
 ```
 
+## CAN Bus
+
+CAN object has a thread parsing the CAN bus it represents. The thread uses the parsing functions of each device in the devices vector. Devices add themselves to the vector when they are passed a CAN object when created.
```

Comment:
Can you give a bit more detail on how the queue system works and why there is nothing a developer needs to do on the CAN driver side of things